### PR TITLE
Update French translations for release 2019.4

### PR DESCRIPTION
**Pull request by njohner**
- Added the French translations for release 2019.4.
- Update translation for disposition archive action

Screenshot 2019-11-25 at 09 27 36

This is for https://github.com/4teamwork/opengever.core/issues/6080

- [x] Changelog-Eintrag vorhanden/nötig?

### CODE REVIEW

File: opengever/core/locales/fr/LC_MESSAGES/plone.po [hunk 1]
```diff
@@ -270,37 +270,37 @@ msgstr "Actif"
...
 #: opengever/core/profiles/default/workflows/opengever_workspace_todolist/specification.txt
 msgid "opengever_workspace_todolist--ROLE--WorkspaceMember"
-msgstr ""
+msgstr "Membre d'équipe"
 
 #. Default: "workspace owner"
 #: opengever/core/profiles/default/workflows/opengever_workspace_todolist/specification.txt
 msgid "opengever_workspace_todolist--ROLE--WorkspaceOwner"
-msgstr ""
+msgstr "Titulaire"
```
Comment (andresoberhaensli) [inline opengever/core/locales/fr/LC_MESSAGES/plone.po]: "Responsable" is commonly used in GEVER for "Federführend".

File: opengever/document/locales/fr/LC_MESSAGES/opengever.document.po [hunk 2]
```diff
@@ -78,7 +78,7 @@ msgstr "Checkout de ${title} n'a pas abouti."
 
 #: ./opengever/document/checkout/checkout.py
 msgid "Could not check out object: ${title}, it is not a document."
-msgstr "${title} n'est pas un document, par consequet checkout n'a pas abouti."
+msgstr "${title} n'est pas un document, par consequent checkout n'a pas abouti."
```
Comment (andresoberhaensli) [inline opengever/document/locales/fr/LC_MESSAGES/opengever.document.po]: Less contrived:
${title} n'est pas un document, le checkout n'a donc pas abouti.

File: opengever/task/locales/fr/LC_MESSAGES/opengever.task.po [hunk 3]
```diff
@@ -1072,7 +1072,7 @@ msgstr "Accompli par ${user}"
 #. Default: "Resolved by ${user} and ${documents} returned"
 #: ./opengever/task/response_description.py
 msgid "transition_msg_resolve_with_documents"
-msgstr ""
+msgstr "Accompli par ${user} et documents renvoyés."
```
Comment (andresoberhaensli) [inline opengever/task/locales/fr/LC_MESSAGES/opengever.task.po]: DE Version appears to list the document count ("${documents}") returned. If that hasn't been implemented in FR, the following would look better:
"Accompli par ${user}, documents renvoyés."

File: opengever/workspace/locales/fr/LC_MESSAGES/opengever.workspace.po [hunk 4]
```diff
@@ -122,17 +122,17 @@ msgstr "Titulaire"
...
 #: ./opengever/workspace/todo.py
 msgid "label_completed"
-msgstr ""
+msgstr "Accompli"
 
 #. Default: "Deadline"
 #: ./opengever/workspace/todo.py
 msgid "label_deadline"
-msgstr ""
+msgstr "A faire d'ici le"
```
Comment (andresoberhaensli) [inline opengever/workspace/locales/fr/LC_MESSAGES/opengever.workspace.po]: Could this be better?
"À compléter d'ici le"

File: opengever/core/locales/fr/LC_MESSAGES/plone.po (hunk 1 above)
Comment (njohner) [reply opengever/core/locales/fr/LC_MESSAGES/plone.po]: Titulaire seems to be used everywhere in Teamraum.

File: opengever/document/locales/fr/LC_MESSAGES/opengever.document.po (hunk 2 above)
Comment (njohner) [reply opengever/document/locales/fr/LC_MESSAGES/opengever.document.po]: Agreed. I had only corrected the typo, but seems faire to change that translation alltogether

**Pull request merged**